src/components/Textarea/styled/textarea.js

```diff
@@ -22,9 +22,8 @@ const StyledTextarea = attachThemeAttrs(styled.textarea)`
     :active {
         outline: 0;
         background-color: ${props => props.palette.background.main};
-        border: solid 2px ${props => props.palette.brand.main};
+        border: solid 1px ${props => props.palette.brand.main};
```

Comment:
we need to have the 2px border when the Textarea is active

Reply:
ok